src/Sentry/Platforms/iOS/SentrySdk.cs

```diff
@@ -6,11 +6,6 @@ namespace Sentry;
 
 public static partial class SentrySdk
 {
-    /// <summary>
-    /// Causes a native crash.  Use for testing purposes only.
-    /// </summary>
-    public static void CauseNativeCrash() => SentryCocoa.SentrySDK.Crash();
```

Comment:
Note: Removing this wasn't a breaking change, because we just recently added it and it was never deployed.